# geneious

Key points

```text

(1) all sequences from each individual and each locus should be combined into a named contig (sample_name_locus_primers). In this case, there were two overlapping amplicons with 3 sequences, and I assembled them into one contig (see attached geneious file). 
(2) consensus sequences should be aligned in a nexus file and the primer sequences should be deleted from the Sanger data (because the primers get included in the amplification and may not exactly match the genome).  
(3) if only one read is obtained, you can still name it as above but also keep the sequence ID (e.g., sample_name_locus_primers_bcn17384) and you can still import it into a nexus file and align it
```

when the results are ready,
1.go to
```
https://genesifter.research.sickkids.ca/gsle/OM/orderList
```

2.Then select the order 
3.select all chromatograms and then download chromats
4.Drag and drop the file to geneious
5.analyze data
